### ISSUE TITLE
Validating New Recipe Returns RECIPE_TYPE_NOT_FOUND Warning

### ISSUE THREAD

**Issue by emimaesmith** [ngageoint/scale#1684]
Creating a new recipe then clicking the validation comes back with a warning 'Cannot find existing recipe type'.  This appears to be coming from the RecipeTypeManager.validate_recipe_type_v6 method (line 1382) where it's attempting to grab the previous recipe type definition to perform a diff. Of course this check will fail on a brand new recipe. This warning does not prevent the recipe type from being pronounced valid or ultimately created, but should not be happening on brand new recipe types.

```
recipe/models/RecipeTypeManager.validate_recipe_type_v6 lines 1382-1400:
            try:
                recipe_type = RecipeType.objects.all().get(name=name)
                old_definition = recipe_type.get_definition()
                diff = RecipeDiff(old_definition, definition)
                if not diff.can_be_reprocessed:
                    msg = 'This recipe cannot be reprocessed after updating.'
                    warnings.append(ValidationWarning('REPROCESS_WARNING',msg))
                    is_valid = False
                json = convert_recipe_diff_to_v6_json(diff)
                diff = rest_utils.strip_schema_version(json.get_dict())
            except RecipeType.DoesNotExist as ex:
                if name:
                    msg = 'Unable to find an existing recipe type with name: %s' % name
                    warnings.append(ValidationWarning('RECIPE_TYPE_NOT_FOUND', msg))
                pass
            except Exception as ex:
                errors.append(ex.error)
                logger.exception('Unable to generate RecipeDiff: %s' % ex)
                pass
```

**Comment by JohnPTobe** [comment]
Names are supposed to be optional for the validate rest endpoint. The create endpoint does not take a name and generates one from the title.  We should update the ui to not ask for a name and not pass it to the validate endpoint when creating a recipe type as the code will ignore the specified name when creating the recipe type anyway.

**Comment by emimaesmith** [comment]
Permanent fix addressed in #1700
